chemper/graphs/cluster_graph.py

```diff
@@ -100,6 +105,36 @@ def add_atom(self, atom):
             """
             self.decorators.add(self.make_atom_decorators(atom))
 
+        def compare_atom(self, atom):
+            """
+
+            Parameters
+            ----------
+            atom: chemper Atom
+
+            Returns
+            -------
+            score: int
+                 how similar is atom to current storage, max of 7 for all decorators identical
+                 0 if atom's atomic number not included in current set
```

Comment:
"0 if atom's atomic number..." also not totally clear. Does this mean that if you have a wildcard atom but with similar decorators it is still 0?

Reply:
So I hadn't thought through the "wildcard" option completely. The input is always an atom, but its true that technically the atom storage (self) could be empty, which right now would cause a problem, I think it makes most sense to give that a high score since if the storage is empty it should match with any atom. I need to think about the best way to handle that scenario.